### PR TITLE
Describe the Interpolate checkbox

### PR DESCRIPTION
**Pull request by will-moore**
As requested by IDR submitter.
cc @sbesson @pwalczysko

### CODE REVIEW

**Comment by pwalczysko** [comment]
I think this makes good sense for the guide and the level of detail into which we want to go here . The second sentence is reassuring.  
Builds well, and the proximity of the screenshot is handy too, see below
Screenshot 2022-04-08 at 11 04 34

**Review by pwalczysko — APPROVED** [review]

**Comment by pwalczysko** [comment]
Will leave @sbesson to comment as well.

**Review by sbesson — COMMENTED** [review]
Thanks, tested in the context of a pyramidal image like http://idr.openmicroscopy.org/webclient/img_detail/1920105/ where you can see the effect of interpolation

Screenshot 2022-04-08 at 11 39 14

Screenshot 2022-04-08 at 11 39 12

When setting the zoom to 100%, no difference is visible. This is inline with my reading of this explanation as the pixel should be fetched from the exact resolution

Screenshot 2022-04-08 at 11 39 20

Screenshot 2022-04-08 at 11 39 25

Some testing with the intensity measurement tool seems to indicate the returned value is independent of this settings. Is that correct? If so it might be worth mentioning here to reduce the anxiety that the setting will modify a intensity result.

**Comment by pwalczysko** [comment]
> Some testing with the intensity measurement tool seems to indicate the returned value is independent of this settings

This matches my observation too

**Comment by jburel** [comment]
@will-moore on return 
Could you address the comment
```
Some testing with the intensity measurement tool seems to indicate the returned value is independent of this settings. Is that correct? If so it might be worth mentioning here to reduce the anxiety that the setting will modify a intensity result.
```

**Comment by will-moore** [comment]
Added a note to confirm that behaviour.

File: docs/iviewer_viewing.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ Step-by-Step
 
    |image0b|
 
-#. The ``Interpolate`` option provides a smoother rendering of the image in the viewer. Unchecking this will show the pixels at the same resolution as the original data.
+#. The ``Interpolate`` option provides a smoother rendering of the image in the viewer. Unchecking this will show the pixels at the same resolution as the original data. This option has no affect on the underlying pixel intensity values.
```
Comment (pwalczysko) [inline docs/iviewer_viewing.rst]: affect -> effect

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**